docs/styles.css

```diff
@@ -0,0 +1,216 @@
+* {
+    box-sizing: border-box;
+    padding:0;
+    margin:0;
+    
+    
+    
+}
+
+body{
+    height: 100vh;
+    background-image: url(assets/bg-desktop.png);
+    background-size: 100vw 50vh ;
+    background-repeat: no-repeat;
+    background-position: bottom;
+    background-color:hsl(229, 57%, 11%); 
+    display:flex;
+    justify-content: center;
+    flex-direction: column;
+    font-family: 'Raleway', sans-serif;
+    font-size: 1rem;
+    
+    
+    
+} 
+   
+.floatContainer{ 
+    height: 170px;
+    display:flex;
+    flex-direction:row;
+    flex-wrap: wrap;
+    justify-content: center;
+}
+
+
+.fyloCard {
+    height:100%;
+    width:300px;
+    display: flex;
+    flex-direction: column;
+    background-color:hsl(228, 56%, 26%);
+    border-radius: 10px 90px 10px 10px;
+    margin-right: 1%;
```

Comment:
using percentage for margin and padding is very risky if you don't have absolute px container

please search and read about: boxed layout vs scratch layout

Reply:
Oh wow, I didn't even notice that was changing when I changed screen size because 1% is so small. Set it to static 10 px now, Thanks.